util.go

```diff
@@ -3,10 +3,31 @@
 package gosnowflake
 
 import (
+	"context"
 	"database/sql/driver"
+	"github.com/google/uuid"
 	"time"
 )
 
+type contextKey string
```

Comment:
Seems unnecessary/premature.

Reply:
One of the linters complained about using bare `string` and it said to make a specific type to avoid collisions

Reply:
GoLint "should not use basic type string as key in context"

Reply:
Oh, interesting. I didn't that of that. Disregard my comment. Thanks linter!